Griddly.Mvc/GriddlyColumn.cs

```diff
@@ -272,7 +272,8 @@ public override object RenderCellValue(object row, bool stripHtml = false)
 
             if (stripHtml && value is string valueString && !string.IsNullOrEmpty(valueString))
                 value = HttpUtility.HtmlDecode(_htmlMatch.Replace(valueString, "").Trim().Replace("  ", " "));
-
+            if (value is IEnumerable<object> coll)
```

Comment:
1) Not sure if we want Griddly to handle this automatically or if it should just be in the template (Joel and Chris please chime in)

2) This doesn't handle cases where the generic collection is of a primitive type (int, decimal, Enum, etc). You could check for the non-generic IEnumerable, but watch out for string (which is IEnumerable<char>)

3) Should the string.Join("," be string.Join(", " (note space after the comma)?

4) To be consistent, should the same logic apply in RenderCell() and RenderUnderlyingValue()

Reply:
What's it currently render as? `[object Object]`? I think we probably want to at least make that saner by default.

Reply:
@ithielnor My concern is, if we support Collections as an expression, do we fully support it? Or are there other things that won't work properly (sorting for instance).

Reply:
Note: Currently sort is already an issue when having non-system objects as the expression. I picked a grid at random in M3 (Memberships/Assessment/AuditsGrid) and it has a .Column(c => c.FeeType) which is a mapped object in NH. Sorting by it sorts by Id instead of by Name (although the name is rendered because that's the behavior of ToString()).

What I'm saying is that by supporting this in Griddly, we're enabling sloppy coding which causes buggy behavior. Case in point above.

Reply:
Sorting is a function of the data access object, not the grid model. `RenderCellValue` comes after the expression resolution. I'm ok with it. It makes it easier to do things like:

```
.Column(x => x.Children, expressionString: "ChildrenWarehouse") // render the actual children, but sort by the warehouse
```

If we want to be overly helpful, we could set expressionString to nothing when the lambda evaluates to a collection too (if we don't already). That would make `.Column(x=> collectionpath)` just render as a non-sortable column.

Reply:
While we're at it, should do the same thing (make it non-sortable) for other unrecognized types.

Reply:
In M3, this is only a problem for custom query output where the user cannot select a property from the objects in the collection. In this case, it only outputs the first object in the collection.